OWNERS

```diff
@@ -14,3 +15,4 @@ reviewers:
   - lluunn
   - willb
   - yph152
+  - johnugeorge
```

Comment:
Glad to get your help!

I think we could add approver, no need to add both approver and reviewer.